src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -460,6 +463,10 @@
       <Generator>MSBuild:Compile</Generator>
       <SubType>Designer</SubType>
     </Page>
+    <Page Include="Views\GuidedTour\ExitTourWindow.xaml">
```

Comment:
Can we rename this to be something else? I think in the future we will also use this area to display other real time information so I would like a generic name, like real time info popup or something else.

Reply:
@QilongTang 
The ExitTourWindow was renamed to RealTimeInfoWindow, see commit: https://github.com/DynamoDS/Dynamo/pull/11918/commits/e6076f46422a6868c3d8e9c36106b11ada9733ec